tests/unit/test_query_retry.py

```diff
@@ -0,0 +1,153 @@
+# Copyright (c) 2021 The PyBigQuery Authors
```

Comment:
Google in this repo: https://opensource.google/docs/releasing/preparing/#Apache-header

Reply:
sorry, fixed